stmp.go

```diff
@@ -28,18 +29,62 @@ func readConfig() {
 	}
 }
 
+type Logger struct {
+	prints chan string
+}
+
+func (l Logger) Printf(s string, as ...interface{}) {
+	l.prints <- fmt.Sprintf(s, as...)
+}
+
 func main() {
+	help := flag.Bool("help", false, "Print usage")
+	enableMpris := flag.Bool("mpris", false, "Enable MPRIS2")
```

Comment:
is there any reason not to enable mpris?

Reply:
It's a pretty naive implementation. The MPRIS2 spec isn't exactly small, and there are several requirements that I'm certain this implementation doesn't satisfy. Also, my understanding of dbus is nebulous and I'm probably not naming things properly. It works well enough with [playerctl](https://github.com/altdesktop/playerctl), but mostly because I beat it into submission. I feel uncomfortable enabling something that I understand as superficially as these protocols. I haven't tested it with any other clients than `playerctl`; it's possible that one of the "required" MPRIS2 elements I've ignored could cause a different client to barf.

That said, I doubt the mpris code will cause stmp to crash, so maybe having it on by default isn't harmful. It's your call -- I just want you to be aware of the caveats.